bin/stt_test/runner.py

```diff
@@ -1,24 +1,25 @@
 import argparse
 import logging
-
+from importlib import import_module
+from .stt_providers.ISTTBase import ISSTBase
 from .utils.test_data import download_test_audio
 
 
-def get_stt_provider(provider_name: str):
-    match provider_name:
-        case "sphinx":
-            pass
-        case "vosk":
-            pass
-        case "whisper":
-            pass
-        case _:
-            logging.error(
-                "Provider not found. Use --help to get available STT providers")
+def get_stt_provider(provider_name: str) -> ISSTBase:
+    try:
+        module = import_module(
+            f"stt_test.stt_providers.{provider_name.lower()}.{provider_name}")
+        return getattr(module, provider_name)()
+    except Exception as e:
```

Comment:
This error most likely will be unreadable at all, it will say something like 'cannot import module ... XYZ'. IMHO we should explicitly declare which providers can be used and not rely on importing modules based on string - it looks very fragile.

Reply:
How about a simple if ... else or a dict lookup and a proper error if no module matches?

Reply:
Sure. I can just use "match"